packages/manager/apps/dedicated/client/app/ip/byoip/disclaimer/disclaimer.html

```diff
@@ -0,0 +1,11 @@
+<oui-modal
+    data-primary-label="{{ ::'ip_byoip_disclaimer_confirm_text' | translate }}"
+    data-primary-action="$ctrl.goToExpressOrder()"
+    data-secondary-action="$ctrl.cancel()"
```

Comment:
You use `cancel` instead of `goBack` for tracking usage ?

Reply:
Yes, this will be followed by a ticket for tracking